extensions/quick-access/package.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quick-access",
+  "title": "Quick Access",
+  "description": "Quick copy, open, reveal the latest files in pinned Finder directories.",
+  "icon": "extension-icon.png",
+  "author": "koinzhang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "pin-directory",
+      "title": "Pin Directory",
```

Comment:
Is `Pin` the right name?

Reply:
The name of this extension was inspired by the Quick Access feature on windows.
So before I set the name to Quick Access and Pin Directory.
Do you have a good suggestion?

![02iOkvupvcqZ5Ybx270ZmGP-28 fit_lim size_1536x](https://user-images.githubusercontent.com/36128970/164712769-54d6d32f-bbc0-4a02-b06b-719aba415649.png)

Reply:
Ah I wasn't aware of this feature. Then let's go with this name and trust Microsoft's naming ;-)